index.html

```diff
@@ -13,12 +13,17 @@
     <title>mobile skeleton</title>
 </head>
 <body>
-    <section>
-    <div class="main-mobile">
+   
+    <section class="main-mobile">
     <header>
         <nav>
             <a href="#"><h2 id="welcome">Welcome</h2></a>
             <div class="nav-links">
+                <ul>
+                    <li><a href="Portfolio">Portfolio</a></li>
+                    <li><a href="About">About</a></li>
+                    <li><a href="Contact">Contact</a></li>
+                </ul> 
```

Comment:
- [ ] As mentioned by the previous reviewer, your nav bar is more towards the left. Please place it at the centre as seen in the Figma template.
### Your Implementation
![image](https://user-images.githubusercontent.com/45460192/186717526-dbf904e0-c975-4bbb-aeb3-6435f14aea25.png)
### Figma
![image](https://user-images.githubusercontent.com/45460192/186717618-13a4fc95-7280-4426-a6d7-f7cc1b1d4d92.png)